client/javaGui/freq-client/pom.xml

```diff
@@ -0,0 +1,261 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.dash.freq</groupId>
+    <artifactId>freq-multimodule</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+  <groupId>io.swagger</groupId>
```

Comment:
Can you change the Ids to be consistent with other pom. `org.dash.freq`
The `description`, `url` and `scm` to match the project URL

Reply:
By project URL are you referring to the Github repo url?

Reply:
Yes.

Reply:
Updated.